app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -3,61 +3,66 @@
 @import 'dropzone';
 
 .tabbable-panes {
-    min-height: 500px;
```

Comment:
Recommend reviewing with whitespace changes hidden (because editor wants this to have two-space indents like everything else).